src/components/dashboard/Dashboard.tsx

```diff
@@ -1,44 +1,39 @@
 'use client'
 import { useState } from 'react'
```

Comment:
We should rename this to `AdminNav`. That would be a better name for this component

Reply:
Will be rolled into next push